shared/teams/team/container.js

```diff
@@ -35,6 +37,15 @@ const mapStateToProps = (state: TypedState, {routeProps, routeState}): StateProp
   isTeamOpen: state.entities.getIn(['teams', 'teamNameToTeamSettings', routeProps.get('teamname')], {
     open: false,
   }).open,
+  publicityMember: state.entities.getIn(
```

Comment:
slightly dangerous. like should pull teamname out into a const and check it explicitly

Reply:
bail and render nothing if its null

Reply:
@chrisnojima Thanks, done.  (Now throws an error which gets caught by the ErrorBoundary.)